### PR TITLE
Use emitMultiple to simplify multiple event cases

### PR DESCRIPTION
**Pull request by leachbj**

### CODE REVIEW

**Comment by ktoso** [comment]
IMO LGTM, but let's ask the author @drewhk :-)

**Comment by patriknw** [comment]
I wrote a blog post about emit before Summer that we have not published yet, just saying. We can still use it here before that one.

File: _posts/2016-09-23-custom-flows-parsing-xml-part-2.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ new GraphStageLogic(shape) with InHandler with OutHandler {
        val coalesced = buffer.toString()
        isBuffering = false
        buffer.clear()
-       emit(out, Characters(coalesced), () => emit(out, other))
+       emitMultiple(out, Iterable(Characters(coalesced), other))
```
Comment (drewhk) [inline _posts/2016-09-23-custom-flows-parsing-xml-part-2.md]: I would use an explicit List here.

**Comment by drewhk** [comment]
LGTM, but use a List instead of Iterable to make the intent clear (there are likely Java readers)

File: _posts/2016-09-23-custom-flows-parsing-xml-part-2.md (hunk 1 above)
Comment (leachbj) [reply _posts/2016-09-23-custom-flows-parsing-xml-part-2.md]: Yeah that's how I've done it in the past, the emitMultiple docs use Iterable (see http://doc.akka.io/docs/akka/2.4.10/scala/stream/stream-customize.html and duplicate example).  Happy to switch it to List() or :: Nil though.

Comment (drewhk) [reply _posts/2016-09-23-custom-flows-parsing-xml-part-2.md]: Docs target Scala devs, while the blog is mixed. I would prefer List even though `:: Nil` is more idiomatic for the same reason :)

**Comment by drewhk** [comment]
Great!

**Pull request merged**